pkg/core/pipeline/target/main.go

```diff
@@ -39,11 +39,11 @@ type Config struct {
 	ReportBody string `yaml:",omitempty"`
 	// ! Deprecated - please use all lowercase `sourceid`
 	DeprecatedSourceID string `yaml:"sourceID,omitempty" jsonschema:"-"`
-	// disablesourceinput disables the mechanism to retrieve a default value from a source.
+	// disablesourceinput disables the mechanism to retrieve a default value from a source. For example, if true, 
+	// source information like changelog will not be accessible for a github/pullrequest action.
```

Comment:
```suggestion
	// disablesourceinput disables the mechanism to retrieve a default value from a source. For example, if true, source information like changelog will not be accessible for a github/pullrequest action.
```

Reply:
According the library used to generate the jsonschema (https://github.com/invopop/jsonschema)
Description is generated based on the line before